.github/workflows/get-secret-from-environment.yml

```diff
@@ -0,0 +1,87 @@
+name: Reusable Fetch Secret Workflow
```

Comment:
This pipeline is used for fetching secrets from different Github Environments. Think for instance staging server provisioning fetching production backup encryption key to be used for periodic backup restore.